lib/-private/todo-handler.js

```diff
@@ -36,7 +36,8 @@ module.exports = class TodoHandler {
 
     let existingTodoFiles = await readTodosForFilePath(this._workingDir, writeTodoOptions.filePath);
 
-    let [, itemsToRemoveFromTodos, existingTodos] = await getTodoBatches(
+    // this should represent the batches - add, remove, stable, expired
+    let [, itemsToRemoveFromTodos, existingTodos, expiredTodos] = await getTodoBatches(
```

Comment:
Let's just call this what it is rather than have a comment here (which doesn't serve much purpose).

```suggestion
    let [, remove, stable, expired] = await getTodoBatches(
```